### PR TITLE
Updating events doc

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
Following changes have been pushed,
1. A note adding change in the default port
and the port range to be used when selinux is in
enforced mode.

2. Console output for `events-api config-get` is
updated.

Fixes: #647
Signed-off-by: srijan-sivakumar <ssivakum@redhat.com>

### CODE REVIEW

File: docs/Administrator-Guide/Events-APIs.md [hunk 1]
```diff
@@ -320,32 +326,32 @@ BRICK_REPLACE                  | volume               | Volume Name
 Event Type                     | Attribute            | Description
 ------------------------------ | -------------------- | -----------
 GEOREP_CREATE                  | force                | Force option used during session Create
-                               | secondary            | Secondary Details(Secondaryhost::SecondaryVolume)
```
Comment (nik-redhat) [inline docs/Administrator-Guide/Events-APIs.md:323]: Why these changes are made here and in other places as well?

Comment (srijan-sivakumar) [reply docs/Administrator-Guide/Events-APIs.md:323]: Strange,,,I think the OR needs to be changed :/ I think my remote repo has to be updated.

Comment (srijan-sivakumar) [reply docs/Administrator-Guide/Events-APIs.md:323]: Ack..issue with my local repo i guess. I think I got into master and pushed rebased from there. :p. Will redo this with main.
